src/services/expenses.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const { DataTypes, Op } = require('sequelize');
+const { sequelize } = require('../db');
+
+const Expense = sequelize.define('Expense', {
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.FLOAT,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+}, {
+  tableName: 'expenses',
+});
+
+function normalize({ id, userId, title, spentAt, amount, category, note }) {
+  return {
+    id, userId, title, spentAt, amount, category, note,
+  };
+}
+
+function getAll() {
+  try {
```

Comment:
It's better to use 'async' and 'await' for error handling in asynchronous functions.